gtsfm/multi_view_optimizer.py

```diff
@@ -23,6 +25,8 @@
 from gtsfm.view_graph_estimator.view_graph_estimator_base import ViewGraphEstimatorBase
 from gtsfm.data_association.dsf_tracks_estimator import DsfTracksEstimator
 
+DEFAULT_OUTPUT_ROOT = Path(__file__).resolve().parent.parent
```

Comment:
Could we instead pass the SceneOptimizer’s output_root property? We already have a default in the runner: https://github.com/borglab/gtsfm/blob/master/gtsfm/runner/gtsfm_runner_base.py#L142

Reply:
Addressed in the recent commit.